Trees Algorithm/binary_tree_traversal_preorder.cpp

```diff
@@ -0,0 +1,21 @@
+//Preorder-> root, left, right
+
+
+#include<bits/stdc++.h>
+using namespace std;
+
+void Preorder( struct node *root)
+{
+  if(root==0)
+  { return 0;}
+ printf("%d", root->data);
+ Preorder( root->left);
+ Preorder( root->right);
+}
```

Comment:
Don't only add the part of the code.
We need a full working code with proper user friendly interface